network/firewall.py

```diff
@@ -0,0 +1,396 @@
+import logging
+import json
+from typing import List, Dict
+
+import network.network as nw
+from nftables import Nftables
+
+logger = logging.getLogger(__name__)
+
+nft = Nftables()
+nft.set_json_output(True)
+nft.set_stateless_output(True)
+nft.set_service_output(False)
+nft.set_reversedns_output(False)
+nft.set_numeric_proto_output(True)
+
+
+class Firewall:
+    firewall_has_been_setup = False
+    postrouting_info: Dict[str, str] = {}
+    forwarding_info: Dict[str, str] = {}
+    vm_info: Dict[int, Dict] = {}
+
+    @classmethod
+    def execute_json_nft_commands(cls, commands: List[Dict]) -> int:
```

Comment:
Why create a class method that does not use anything from the class instead of a standalone function ?